app/views/admin/_button.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_with url: url, method: method do |f| %>
+    <%= f.submit button_text %>
+<% end %>
```

Comment:
Nice use of partials! I wonder, though, whether the number of local variables you have to pass in for such a small partial makes this maybe not worth the effort. Partials are great, but you can really see their power when you use them to replace a more complex form with multiple fields